cypress/integration/sign_in.spec.js

```diff
@@ -0,0 +1 @@
+/// <reference types="Cypress" />
```

Comment:
Do you really need this empty spec file? 🙂

Reply:
Maybe it is not good to leave this file empty for now, but it will be updated soon. :)

Reply:
I guess the file can be easily added as soon as it's needed.
I'd use camelCase for file naming by the way. 🙂